tests/phpunit/unit/BoltUnitTest.php

```diff
@@ -63,11 +63,11 @@ protected function getApp($boot = true)
             $this->app->initialize();
 
             $verifier = new Config\Validation\Validator($this->app['controller.exception'], $this->app['config'], $this->app['resources']);
-            $verifier->checks();
 
             if ($boot) {
                 $this->app->boot();
             }
+            $verifier->checks();
```

Comment:
I'm still laughing at my own stupidity over that one … thanks for catching it :smile: